compiler/rustc_resolve/src/errors.rs

```diff
@@ -785,15 +785,6 @@ pub(crate) struct ItemsInTraitsAreNotImportable {
     pub(crate) span: Span,
 }
 
-#[derive(Diagnostic)]
-#[diag(resolve_is_not_directly_importable, code = E0253)]
```

Comment:
This renders E0253 unused. You should add the following notice to compiler/rustc_error_codes/src/error_codes/E0253.md:

```md
#### Note: this error code is no longer emitted by the compiler.
```

Als0, you need to update the `compile_fail,E0253` code block attributes to `ignore (no longer emitted)` but CI will tell you that anyway.